activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -65,7 +65,6 @@ def deprecated_method_warning(method_name, message = nil)
 
         def deprecation_message(callstack, message = nil)
           message ||= "You are using deprecated behavior which will be removed from the next major or minor release."
-          message += '.' unless message =~ /\.$/
```

Comment:
@jeremy done. Now making sure the sentence ends with '.' lies at where ever its defined.
Before https://github.com/rails/rails/commit/8fc97d198ef31c1d7a4b9b849b96fc08a667fb02, we relied on '.' being present in sentences.